src/api/Dockerfile.frontend-base

```diff
@@ -8,6 +8,9 @@ RUN npm install -g jshint
 # Ensure there are ruby, gem and irb commands without ruby suffix
 RUN for i in ruby gem irb; do ln -s /usr/bin/$i.ruby2.5 /usr/local/bin/$i; done
 
+# Path where gems will be installed. It can be cached with volumes
+ENV BUNDLE_PATH /gems
```

Comment:
Better use /usr/lib/obs-api and use obs-bundled-gems

Reply:
We don't use packages in the development environment...

Reply:
You don't have to - it's enough if the container does.

Reply:
What?